site/content/en/docs/overview/_index.md

```diff
@@ -35,6 +35,26 @@ A core design principle for Kueue is to avoid duplicating mature functionality i
 - **All-or-nothing with ready Pods:** A timeout-based implementation of [All-or-nothing scheduling](/docs/tasks/manage/setup_wait_for_pods_ready/).
 - **Partial admission:** Allows jobs to run with a [smaller parallelism](/docs/tasks/run/jobs/#partial-admission), based on available quota, if the application supports it.
 
+## Job-Integrated Features
```

Comment:
The row names don't necessarily match a feature gate. Maybe that's your intention, but then why are all of them CamelCase as if they were a feature gate?

Reply:
It might also be worth linking each name to a section in the documentation that explains what this is.

Reply:
> The row names don't necessarily match a feature gate. Maybe that's your intention, but then why are all of them CamelCase as if they were a feature gate?

You right, table have not only feature gates. Do you think to rename feature names on table ?

Reply:
> It might also be worth linking each name to a section in the documentation that explains what this is.

Done